app/controllers/report_controller/reports/editor.rb

```diff
@@ -1519,10 +1519,8 @@ def set_form_vars
 
     # Only show chargeback users choice if an admin
     if admin_user?
-      @edit[:cb_users] = {}
-      User.all.each { |u| @edit[:cb_users][u.userid] = u.name }
-      @edit[:cb_tenant] = {}
-      Tenant.all.each { |t| @edit[:cb_tenant][t.id] = t.name }
+      @edit[:cb_users] = User.all.each_with_object({}) { |u, h| h[u.userid] = u.name }
+      @edit[:cb_tenant] = Tenant.all.each_with_object({}) { |t, h| h[t.id] = t.name }
     else
```

Comment:
@isimluk : I know that is should most likely by a separate PR but: Could you refactor this to use `pluck`?
I mean do a projection on the base relation instead of fetching all the columns of the 2 relations here.

I a separate commit, please?